probe-rs/src/debug/unit_info.rs

```diff
@@ -621,6 +621,7 @@ impl<'debuginfo> UnitInfo<'debuginfo> {
                             cache.remove_cache_entry(namespace_variable.variable_key)?;
                         }
                     }
+                    gimli::DW_TAG_formal_parameter |  // Parameters to functions.
```

Comment:
The comments are not aligned anymore. I would not align them either way but a mix is kind of odd :)